bundle/edu.gemini.itc.web/build.sbt

```diff
@@ -27,6 +27,7 @@ OsgiKeys.exportPackage := Seq()
 
 OsgiKeys.importPackage := Seq(
   "!com.google.inject.cglib.asm.util.*",
+  "!com.keypoint.*",
```

Comment:
Why do we need to explicitly not import these packages? It seems they are not used in the code anyway